src/Footer.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const arrAddress = [
+  {
+    address: "123 Fake Street, London, E1 4UD",
+    email: "hello@fakehotel.com",
+    tel: "0123 456789"
+  }
+];
+
```

Comment:
If you read the instructions for exercise 4 carefully, you'll see that it asks you to pass this data *as a prop* to the `Footer` component. Could you change this component so it does that?